src/components/shared/MembersList/SortingRow/SortingRow.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import classnames from 'classnames';
+
+import Button from '~shared/Button';
+import Icon from '~shared/Icon';
+
+import { SortingMethod } from '~gql';
+
+import styles from './SortingRow.css';
+
+export interface Props {
+  sortingMethod: SortingMethod;
+  handleSortingMethodChange: React.Dispatch<
+    React.SetStateAction<SortingMethod>
+  >;
+}
+
+const displayName = 'MembersList.SortingRow';
+
+const MSG = defineMessages({
+  permissions: {
+    id: `${displayName}.permissions`,
+    defaultMessage: 'Permissions',
+  },
+  reputation: {
+    id: `${displayName}.reputation`,
+    defaultMessage: 'Reputation',
+  },
+});
+
+const SortingRow = ({ handleSortingMethodChange, sortingMethod }: Props) => {
+  const nextSortingByRepMethod =
+    sortingMethod === SortingMethod.ByHighestRep
+      ? SortingMethod.ByLowestRep
+      : SortingMethod.ByHighestRep;
+  const nextSortingByRoleMethod =
+    sortingMethod === SortingMethod.ByMorePermissions
+      ? SortingMethod.ByLessPermissions
+      : SortingMethod.ByMorePermissions;
+  const isSortingByRep =
+    sortingMethod === SortingMethod.ByHighestRep ||
+    sortingMethod === SortingMethod.ByLowestRep;
+  return (
+    <div className={styles.container}>
+      <Button
```

Comment:
There's some duplication here between the buttons. Could be encapsulated into one component.

Reply:
Creating SortingButton component